src/coredump/coredump.c

```diff
@@ -153,6 +157,8 @@ static int parse_config(void) {
                 { "Coredump", "JournalSizeMax",   config_parse_iec_size,          0, &arg_journal_size_max  },
                 { "Coredump", "KeepFree",         config_parse_iec_uint64,        0, &arg_keep_free         },
                 { "Coredump", "MaxUse",           config_parse_iec_uint64,        0, &arg_max_use           },
+                { "Coredump", "RateLimitInterval", config_parse_sec,              0, &arg_ratelimitinterval },
```

Comment:
RateLimitIntervalSec please.

please reindent the table, so all rows neatly align again